PROCESSORS.md

```diff
@@ -684,15 +684,17 @@ In the list below, the names of required properties appear in bold. Any other pr
 |Disable Peer Verification|false||Disables peer verification for the SSL session|
 |HTTP Method|GET||HTTP request method (GET, POST, PUT, PATCH, DELETE, HEAD, OPTIONS). Arbitrary methods are also supported. Methods other than POST, PUT and PATCH will be sent without a message body.|
 |Include Date Header|true||Include an RFC-2616 Date header in the request.|
+|invokehttp-proxy-password|||Password to set when authenticating against proxy|
+|invokehttp-proxy-username|||Username to set when authenticating against proxy|
+|Penalize on "No Retry"|false||Enabling this property will penalize FlowFiles that are routed to the "No Retry" relationship.|
 |Proxy Host|||The fully qualified hostname or IP address of the proxy server|
 |Proxy Port|||The port of the proxy server|
+|Put Response Body in Attribute|||If set, the response body received back will be put into an attribute of the original FlowFile instead of a separate FlowFile. The attribute key to put to is determined by evaluating value of this property.|
 |Read Timeout|15 secs||Max wait time for response from remote service.|
 |Remote URL|||Remote URL which will be connected to, including scheme, host, port, path.<br/>**Supports Expression Language: true**|
+|send-message-body|true||If true, sends the HTTP message body on POST/PUT/PATCH requests (default).  If false, suppresses the message body and content-type header for these requests.|
```

Comment:
I would prefer to have the version with the capitals and spaces here and in the config, since this is facing the user and most of our properties are like that.

Reply:
The version with capitals and spaces is just the "display name" of the property and cannot be used in a configuration, so I would not use it in the documentation. We could rename it, but even though it did not have any functionality behind it before, the property was available and could break some previous flows using it in the configuration.

Reply:
I think it's worth renaming all of these properties while we are still in a 0.x version and we can do it.  "Send Message Body", "Proxy Username", "Proxy Password" would match NiFi and it would be more consistent internally in MiNiFi, too.  Currently, we have "Proxy Host", "Proxy Port", but "invokehttp-proxy-username", "invokehttp-proxy-password", which is confusing.

Reply:
I agree it would be more consistent and more intuitive to use those names. If property renaming is not a problem under version 1.0 then I'll change these properties.

Reply:
Update in 54c835ee681bea18cc7172b309181d56e5ab5439

Reply:
Updated in 321d274fd111d0fc64eff092df50fcbb68e6acd7 by deprecating the old property in favor of the new `Send Message Body` property.